src/FilterButtonList.js

```diff
@@ -50,57 +61,99 @@ let FilterButtonItem = _.flow(
                 }}
               />
             </div>
-            <Button onClick={() => tree.clear(node.path)}>Clear</Button>
-            <Button primary onClick={F.off(modal)}>
-              Done
-            </Button>
-          </div>
+            <Flex style={{ justifyContent: 'flex-end' }}>
+              <Button onClick={() => tree.clear(node.path)}>
+                Clear Filter
+              </Button>
+              <Button
+                primary
+                onClick={() => {
+                  F.off(modal)()
+                  tree.mutate(node.path, { paused: true })
+                }}
+                style={{ marginLeft: '10px' }}
+              >
+                Done
+              </Button>
+            </Flex>
+          </Flex>
         </Modal>
-      </>
+      </div>
     )
   }
 )
 
-let GroupBox = ({ nodeJoinColor, children, nested, className }) => (
+let GroupBox = ({ nodeJoinColor, children, nested, className, style }) => (
   <Flex
     wrap
     className={`${className} ${nested ? 'nested' : ''}`}
     alignItems="center"
-    style={{ borderColor: nodeJoinColor }}
+    style={{ borderColor: nodeJoinColor, ...style }}
   >
     {children}
   </Flex>
 )
 
-let FilterButtonList = ({
-  node,
-  tree,
-  fields = {},
-  mapNodeToProps = _.noop,
-  className = 'filter-button-list',
-  nested = false,
-}) => (
-  <GroupBox
-    {...{ nested, className }}
-    nodeJoinColor={node && styles.joinColor(node)}
-  >
-    {_.map(child => {
-      let Component = child.children ? FilterButtonList : FilterButtonItem
-      return (
-        <Component
-          key={child.path}
-          nested
-          {...{
-            tree,
-            node: child,
-            fields,
-            mapNodeToProps,
-            className,
-          }}
-        />
+let FilterButtonList = observer(
+  ({
+    node,
+    tree,
+    fields = {},
+    mapNodeToProps = _.noop,
+    allowDuplicateFields = false,
+    className = '',
+    nested = false,
+    style,
+    children,
+    theme: { Icon, Button, FilterButton = Button },
+  }) => {
+    let options = fieldsToOptions(fields)
```

Comment:
This and the lines below are likely duplicated from the regular filter list - we should probably factor out a reusable utility to get field options that takes allow duplicates as a flag)